### DISCUSSION TITLE
q about documentation site

### DISCUSSION THREAD

**Discussion by fgtrzah** [bcakmakoglu/vue-flow#1232] [Q&A] [answered]
I was not sure where in the docs site repo to look to find out more about the implementation for the cursor animation used in the vue-flow site? I thought it was neat and wanted to utilize something similar for a project and cite vue-flow as the inspiration

**Comment by bcakmakoglu** [comment, marked as answer]
[I'm using `Blobity` for the cursor effect.](https://blobity.dev/)

**Reply by fgtrzah** [reply]
thnk
